lib/components/NavigationMenu/MenuSeparator.tsx

```diff
@@ -0,0 +1,3 @@
+export const MenuSeparator = () => {
+  return <span className="ds:my-3 ds:border-[#CCC] ds:border-b-[1px]"></span>;
```

Comment:
`#CCCCCC`
Color not available in design tokens. Will need update when those are updated or take some other token to use in here.